### DISCUSSION TITLE
Who's hiring (Aug 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#39273] [Ideas]
# What is this?
New month, new post! This discussion is for companies looking for people with Next.js experience. Both permanent positions and contract work are welcome.

# Companies
Please use the following format followed by a brief description of the role:

> COMPANY_NAME | ROLE | LOCATION | Full-time | $100-150k base | LINK

Please only post if you personally are part of the hiring company—no recruiting firms or job boards. Only one post per company.

Good luck with recruiting!

## Sample Job Post

Here's a good example job post.

> Vercel | Senior Full-Stack Engineer | Remote (US, Canada, Europe, or Asia) | Full-time | $100-225k base | [Link](https://vercel.com/careers/senior-full-stack-engineer-amer)
>
> Vercel’s hiring across the board on various engineering teams and positions, but the above link is for our full-stack engineering position. Vercel is innovating on and for the web and at the edge. We build tools and workflows developers love and deliver outstanding performance for end users.
>
> I've been at the company for 2 years and still find plenty of opportunity for impact, and love the teammates I'm working with. Feel free to apply using the link above or reach out to me directly.

---

*If you're looking for last month's discussion, see #38458. At least one company has [hired someone](https://twitter.com/Nutlope/status/1475929984174895105) as a result of it!* 

If you hire someone through this job board, please reach out to me (hassan@vercel.com) and let me know so I can send you some swag :)

If you're a developer looking to post about themselves, use #39274.

**Comment by GYusufu** [comment]
Happy Wednesday, We're Hiring! 😄 

Herff Jones | Sr UI Developer| REMOTE (US) | Full-time/W2 | $100-135k base | https://www.herffjones.com/about/careers/job-openings/?gnk=job&gni=8a7885ac81458258018163cca0844729

HTML DOM, Canvas API, Angular, Ngrx, React...This role is fully remote, we offer comprehensive benefits, PTO, Paid holidays, etc... There is plenty of opportunity for growth and exposure to new technologies within this role.

Feel free to apply using the link above or reach out to me directly concerning this role, another role on the site, or curiosity of the roles we have across our other business units under our parent company, Varsity Brands. Happy hunting, I look forward to speaking with you!

**Reply by Babbili** [reply]
Thank you, i sent my resume by email for related vacancies

**Comment by AlessandroVol23** [comment]
Hashnode | Fully REMOTE & Async | Full-Time

Multiple roles (Engineering, Design, Product)

Hashnode is the #1 blogging platform for developers that allows you to publish articles on your own domain and helps you stay connected with a global developer community. You own your content!

The main tech role is:

Senior Fullstack Developer: Develop the features for the next-gen developer platform. -> https://careers.hashnode.com/jobs/lg_F08w6be30/software-developer-full-stack-javascript-remote
  
The tech-stack:
  - Fully deployed on Vercel
  - Next.js, React, Tailwind
  - TypeScript 
  - MongoDB
  - AWS Serverless - CDK, Lambda, SQS, DynamoDB, Api Gateway, Eventbridge
We love an open writing culture & transparency so you can share with the world what you are working on.
Hashnode is hiring remote all over the globe (not just US only). We work async, which means you set your own schedule working.

We also have a growing [engineering blog](https://engineering.hashnode.com) if you're interested in some topics in more detail.

Hashnode is backed by many great VCs (e.g. Salesforce Ventures, Accel Partners, Sierra Ventures, Sequoia Capital's Surge) and individual investors like Vercel's CEO Guillermo Rauch, AngelList's founder Naval Ravikant, Intercom's founder Des Traynor, Ed Roman, and Salil Deshpande.

Contact me: at sandro@hashnode.com / twitter.com/sandro_vol for more info or check out the careers page: https://careers.hashnode.com/jobs

Contact me for any questions I am happy to help :)

**Comment by nirmpate** [comment]
HOWL | SENIOR SOFTWARE ENGINEER | US REMOTE | Full-time | [LINK](https://grnh.se/16d9af072us)


NOTE: US BASED CANDIDATES ONLY

About Howl

Howl is the creator platform for social commerce. We build tools for brands and creators to sell more products across every social platform, together.

In the last 10 years, the $20bn influencer ecommerce market has become a critical part of the retail industry. Social commerce is a new $1.2 trillion market opportunity ([link](https://www.accenture.com/us-en/insights/software-platforms/why-shopping-set-social-revolution?c=acn_glb_socialcommercebusinesswire_12755585&n=mrl_0122)) and requires new tools, protocols, and technologies.

Howl allows creators and brands to run experiments on what products to talk about and how best to talk about them. Paired with data delivered at the speed of social, creator financing for products, and feedback loops delivered in a beautifully designed platform, Howl takes the guesswork out of selling products on social platforms.

Howl’s mission is to reduce barriers for creators and brands to build social commerce businesses. We are a remote-first company with a diverse team that reflects our vision. Our beta launched in mid 2022 and powers social commerce for creators and brands across beauty, gaming, home, fashion, and consumer electronics for both industry leaders like Best Buy, Samsung, Target and cult favorites like Cariuma, Kosas, and Youth to the People. We are a Series A company backed by investors including HighlandX and NEA.

For more information, visit [www.planethowl.com](http://www.planethowl.com/).

The position
Develop innovative algorithms for real time, high throughput, low latency marketplace system
Successful experience designing and building large, scalable and complex distributed systems
Expertise in the Big Data Technology Stack
Ability to establish strong relationships with product and business stakeholders across the organization to co create objectives and facilitate their own cross-team scope and effort
Effective relationship builder throughout the engineering organization to refine initiatives and requirements, guide design and implementation, articulate outcome and impact
Creates & structures documentation for large code projects concisely & consistently
Ability to mentor teammates across teams based on experience & expertise
Relied upon across the engineering organization to keep complex projects on-track and sustainably implemented
Contribute to refinement and enrichment of development tools and processes
Ability to write persuasive analysis proposals to help make high-quality decisions about priorities among tech debt and tech strategy. Understands technical debt as a long-term budgeting and risk management tool; Can articulates when to incur more and when to pay it down.
Experience
A successful candidate will have substantial relevant experience in high-tech software development teams.
Experience in AdTech or Ecommerce. FinTech is OK too.
Experience in Big Data. Ideally, strong knowledge of ELT, Reverse ETL, DBT, Snowflake, Spark,
Experience in Go and/or C++ and/or Python and/or Java programming and AWS

Education expectation
A bachelor’s or better degree in computer science or software engineering -or-
A bachelor’s or better degree in a STEM discipline plus substantial software engineering experience

Howl Tech Stack
One of the most frequent questions asked by candidates of Howl engineers is, “What is your tech stack?” This is a brief overview of our tech stack.

Howl’s systems are implemented as a modern microservices architecture running on Linux servers hosted in AWS. We use Kubernetes to manage our containers, we use Flask to construct our Web interfaces. We build interactivity in our web interfaces using Nextjs. We use Linux, and in particular Debian, Ubuntu, and Alpine distros.

Our favorite programming languages are Python 3, Scala, Go, Elixir, and, of course, TypeScript and JavaScript. We stash our code in GitHub.

We test each language with an appropriate unit testing tool - JUnit, PyTest, ScalaTest, ExUnit, and Jasmine. We use Jenkins and Github to run our builds and tests.

Check-ins against the main branch require a code review by another SWE.

We keep some data in Snowflake, some in Postgres, others in DynamoDB, and some, of course, in MySQL. We build our product graph using Elasticache. We are working to reduce the number of databases we rely on.

We stream data around using Kafka, Spark, and Storm. Maybe we will prune this list sometime.

We are not hostile to commercial software - we use AirTable, DataDog, Fivetran, Jira, LaunchDarkly, Looker, Matomo, and many more.

https://grnh.se/16d9af072us

**Comment by s1nny0ur** [comment]
Highs And Lows started in 2005 as a counter-cultural reaction to the retail offering in Australia at the time. For fifteen years, Highs And Lows has continued to celebrate underground culture, curating important brands from throughout the world.
                
As an independently owned business we are a tight knit team of driven and passionate people who push for the highest standards each day, we are afforded a lot of freedom to follow new ideas and pursue creative endeavours within a growing business and brand.      
We are always on the lookout to add to our growing community of designers, artists, photographers and collaborators, our goal is to continue creating meaningful projects that push the boundaries of taste and influence.
                 
Highs and Lows is committed to promoting an inclusive culture and freedom from discrimination, as an equal opportunity employer we value our team and find nourishment in a workplace that is respectful and peaceful.
                   
Our Retail operations based in Perth Western Australia have operated for over 15 years. If you are interested in joining the team please hit us up via the emails provided below.


_Available Roles:_

**Front-end developer ( Remote / Freelance )** -  You deliver solid solutions to complex problems and can self manage to provide consistent, high-quality code delivered on time.

**As a front end developer, you’ll be:**
Developing high quality solutions, both big and small.
Building working and innovative solutions 
Working with designers and product teams to bring UX prototypes to life.
Working with backend engineers to integrate the front end with APS, CMS layouts, etc – the integration will vary based on the project.

Working with project management to plan, estimate and divide work amongst the team.


**Role Requirements:**
A passion for JavaScript development
An understanding of critical CSS techniques (TailwindCSS, CSS in JS, styled components, CSS modules etc)
Experience in building reusable frontend components and experience with design systems is a plus.
Working experience with Next.js and deploying to Vercel.
Proficient understanding of JavaScript (ES7+)
Working experience with compilers e.g. Webpack, Grunt or Gulp.
Experience with Node.JS servers (Express JS or similar)
Experience with unit testing frameworks (jest or similar, storybook).
Cross-browser compatibility experience for all major browsers
Cross device compatibility experience for all major operating systems and device types (iOS, Android, Windows)
Understanding of internet security issues and mitigation strategies
Solid understanding in building accessible components and websites
Fantastic attention to detail.
Self-motivated with the ability to work independently, or with a team.
Team player with excellent collaboration skills.

**Additional Desired Skills:**
Experience with TypeScript
Experience with Web Components
Experience in working with CMS’s like Sanity, Prismic or Contentful.
Experience with cloud technologies like AWS and/or Google GCP
Willingness to learn new things and teach others what you know
Strong written and verbal communication skills
Strong presentation and group facilitation skills


Please send relevant visuals/portfolio with a covering letter to: dev.careers@highsandlows.net.au

**Comment by ShneorCr** [comment]
Hey  
We're looking to hire a senior Nextjs+Supabase (Nodejs too) developer for a long-term project for the healthcare industry.

Please let me know if you have the right experience and are available to join our remote team immediately.

**Reply by arsantiagolopez** [reply]
Hi Shneor!

I'm a full stack TypeScript developer with about 4 years of experience on Next.js, Node.js and Supabase. If it's not too late, I'd love to hear more about the project. I'm able to join immediately.

Feel free to reach me at [arsantiagolopez@gmail.com](mailto:arsantiagolopez@gmail.com).

Best,
Alex

**Reply by smithsoft99** [reply]
Hello. Shneor.
Hope you are doing well.
Thank you for your job posting.
I am sure I have strong skills and rich experience that you are looking for.
This is my website.
https://alexsmith.herokuapp.com/
I look forward to hearing from you soon.
Warm regards.
Alex

**Comment by BLevesqueSTM** [comment]
SmarterTravel | Senior Infrastructure Engineer | Remote (US) | Full-time | https://www.wearesmartertravel.com/smartertravel-careers/?4272339?gh_jid=4272339 

SmarterTravel is looking for an individual who can leverage their DevOps and Software Engineer skills to help us improve processes and scale. You can expect to build, deploy, and maintain a core set of AWS services, and support data warehousing, monitoring, and alerting. The tech stack/tools includes Python, GitHub Actions, Terraform, Redshift, and Kubernetes.

If you're someone who likes knowledge sharing and supporting teams cross-functionally, I'd love to set up some time to chat and tell you more about the role. Please reach out at blevesque@smartertravel.com

**Comment by IraES0116** [comment]
EQUUS SOFTWARE| SOFTWARE ENGINEER (FRONT END) | LISBON | Full-time | https://apply.workable.com/equus-software/j/32DB727635/

As a Software Engineer, you will be building out key parts of our framework
that is used to build new products.

You will work with self-organizing development teams to improve the team,
process, product and user experience. Day-to-day, you will collaborate with
your team of developers, business analysts and product owners. You will also
work with subject matter experts from other teams as well as
implementation, support staff and sometimes Clients to produce a steady
stream of new features

Requirements
WHAT YOU BRING
3+ years proficiency with at least one object-oriented programming language (C#, Java, etc.)
Experience with Javascript SPA like Angular/React, Angular 2, TypeScript, JQUERY, Twitter Bootstrap, JavaScript, CSS, HTML
Understanding of complex concepts, such as recursion, reflection, multi-threaded processes, memory management
Proven record of leading and successful delivery of multiple projects
Experience working in both individual and team environments
NICE TO HAVE
4+ years of experience operating in an agile environment (Scrum or Kanban)
Experience with ASP.NET MVC
Experience with Blazor
Test Driven Development using a testing framework like MSTest or NUnit
Experience using an ORM framework like NHibernate or Entity
Framework
Object-oriented design
Database design
T-SQL
Code and database performance analysis
Secure coding practices
Troubleshooting complex problems
Visual Studio
SQL Server Management Studio
UI design
Benefits
Permanent Role – You will not start over when moving projects. We only hire passionate people who are looking for a long-term opportunity with a growing company.
We LOVE your ideas! Innovation Day, once a quarter you can share your fresh ideas.
Diversity/Inclusion - With employees from far-reaching countries across the globe, we've welcome unique mindsets and different backgrounds.
Culture – In Equus the work life balance is valued. For that reason, we try to organize team outings and team buildings activities regularly.
Salary & Benefits - We have highly competitive salaries with an exceptional benefit package, that includes medical coverage, PTO, flexible work hours, plus more.
Great people, great leaders. We only hire the BEST in the market so they can become the best professionals of industry! Our people are always available to help one another.
Yearly team events where you’ll be able to meet all of your colleagues face to face in one place.

**Comment by smithsoft99** [comment]
Hi,
Thank you for your message.
First of all, I sincerely hope you are doing well and safe from covid-19.
I am very interested in this opportunity.
This is my resume.
I look forward to hearing from you soon.
If I missed your call, please send me a message via email.
Warm regards.
Alex
